Tools/parametric_model/generate_parametric_model.py

```diff
@@ -94,6 +97,26 @@ def start_model_estimation(config, log_path, data_selection=False, plot=False):
         from visual_dataframe_selector.data_selector import select_visual_data
         model.data_df = select_visual_data(model.data_df,visual_dataframe_selector_config_dict)
         model.n_samples = model.data_df.shape[0]
+    if auto_data_selection:
+
+        idx = model.data_df.sort_values(by=["fisher_information_force"],ascending=False).index[0:model.data_df.shape[0]*10//100]
+        idx = idx.append(model.data_df.sort_values(by=["fisher_information_force"]).index[0:model.data_df.shape[0]*10//100])
+        idx = idx.append(model.data_df.sort_values(by=["fisher_information_rot"],ascending=False).index[0:model.data_df.shape[0]*10//100])
+        idx = idx.append(model.data_df.sort_values(by=["fisher_information_rot"]).index[0:model.data_df.shape[0]*10//100])
```

Comment:
Where does `10//100` actually come from?

Reply:
The current implementation of the automatic data selection chooses the top 10% observations according to the information metric. Currently it does not work as reliable as I hoped it would.

Reply:
Thanks for clarifying. Could you maybe write this down in the comments so that we dont forget what is going on here?

Reply:
I'm on it.

Reply:
Danke Pascal!